content/azure-boards-tips-operations-teams.md

```diff
@@ -0,0 +1,100 @@
+Title: Azure Boards Tips for Operations Teams
+Date: 2022-12-23
+Category: Posts 
+Tags: azure-devops, tips
+Slug: azure-boards-tips-operations-team
+Author: Willy-Peter Schaub
+Summary: A few useful "did you know" tips when using Azure Boards for Operational-type teams.
+
+After observing users work with and occasionally struggle with the [Azure Boards](https://azure.microsoft.com/en-ca/products/devops/boards/), it is time for me to share a few tips. We continue from [Agile Boards Tips](/agile-boards-tips.html) and focus on operational teams this time.
+
+---
+
+# What makes an Operations team different from a Delivery team?
+
+There are likely tons of definitions and views of groups of engineers making up an operations, operational, services, or system teams. I will focus on my team, Common Engineering, is one of our common services teams, focused on operations. We empower our delivery teams, aka software development, with consistent and standardized **processes** and **products**, **innovation** of tool chains and continuous delivery pipelines, and **support** of operational infrastructure.
+
+> ![Intake](../images/azure-boards-tips-operations-teams-1.png) 
+
+As shown, the focus on processes, products, innovation, and support are creating a  variation of support requests we receive and associated work we plan after triage and prioritization. A challenging, but exciting mix of services we are focused on every day. 
```

Comment:
requests we receive and associated work we plan after triage and prioritization
consider:
requests we receive and associated work that we plan after triage and prioritization